cmake/Modules/FindMKLDNN.cmake

```diff
@@ -122,9 +122,6 @@ IF(NOT TARGET mkldnn)
   MESSAGE("Failed to include MKL-DNN target")
   RETURN()
 ENDIF(NOT TARGET mkldnn)
-IF(MKL_FOUND)
-  TARGET_COMPILE_DEFINITIONS(mkldnn PRIVATE -DUSE_MKL)
```

Comment:
@bddppq
disabled this flag for MKLDNN, which caused the build failure.
This flag didn't impact on pytorch any more.

Reply:
Does disabling MKL have perf implications on MKLDNN?

Reply:
according to below table from MKLDNN, should be no perf change.
Anyway, we use jit impl.

  /* USE_MKL      USE_CBLAS       effect
   * -------      ---------       ------
   * yes          yes             use Intel(R) MKL CBLAS
   * yes          no              use jit
   * no           yes             system-dependent CBLAS
   * no           no              use jit
   */